rest-model/src/main/java/com/gentics/mesh/core/rest/node/field/S3BinaryField.java

```diff
@@ -11,22 +11,22 @@
      *
      * @return
      */
-    String getS3BinaryUuid();
+    String getS3binaryUuid();
 
     /**
      * Set the Uuid of the used s3binary data.
      *
      * @param uuid
      * @return
      */
-    S3BinaryField setS3BinaryUuid(String uuid);
+    S3BinaryField setS3binaryUuid(String uuid);
 
     /**
      * Return the s3binary filesize.
      *
      * @return Filesize in bytes
      */
-    long getFileSize();
+    Long getFileSize();
```

Comment:
Why use boxing here?

Reply:
I am using it here since filesize is not required to be inserted in the Mesh immediately and by using a long primitive, i was actually inserting a field filesize=0. I saw that in many fieldtypes the Number Classes are preferred. Let me know if I should change it.